### PR TITLE
Show player details for match lineups

### LINKED ISSUE #57: Show additional info on match pages
Show the additional player info like on the team page and calculate the average elo for the announced lineup.
Use the same table approach as discussed in #60 .

image

### PR DESCRIPTION
**Pull request by timonla**
* Also fixed the member tables to always show all members

image

Resolves #57

### CODE REVIEW

**Comment by Lyannic** [comment]
Do we really want to mirror the table for the other team?

**Review by Lyannic — CHANGES_REQUESTED** [review]

File: src/components/FaceitLogo.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import { customElement, property } from 'lit/decorators.js';
+
+import { FaceitInfo } from '../model';
+import { getFaceitLevelLogoLink, getFaceitLink } from '../util/getLink';
+
+const flashkillFaceitLogo = 'flashkill-faceit-logo';
+
+@customElement(flashkillFaceitLogo)
+class FaceitLogo extends LitElement {
+  @property({ type: Object }) faceitInfo!: FaceitInfo;
```
[the remark this answers is not in the document]
Comment (timonla) [reply src/components/FaceitLogo.ts]: Fixed it

File: src/components/LineupTables.ts [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+            header="Profiles"
+            auto-width
+            text-align="center"
+            .renderer="${this.profilesRenderer}"
+          ></vaadin-grid-column>
+          <vaadin-grid-column
+            header="Name"
+            auto-width
+            text-align="end"
+            .renderer="${this.nameRenderer}"
```
[the remark this answers is not in the document]
Comment (timonla) [reply src/components/LineupTables.ts]: Decided to go with a non-mirrored version, as you suggested in another comment. Was able to reuse the table creation like that.

image

File: src/components/LineupTables.ts [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+import './FaceitEloBar';
+import './ToggleContentButton';
+import './PlayerProfiles';
+import { customTheme } from '../util/theme';
+import { FaceitInfo, LineupPlayer } from '../model';
+import { get99PlayerLink } from '../util/getLink';
+import { avgFaceitElo } from '../util/avgFaceitElo';
+
+export interface LineupTableItem {
```
Comment (Lyannic) [inline src/components/LineupTables.ts:17]: You don't need to change this, just a remark. I don't like the technical naming that you apply here. A lineup probably consists of players or something like that. You are also using very technical names for the props (`lineupItems1`). What I would expect, is just a lineup. So `lineup1` and `lineup2` as props.

**Review by Lyannic — APPROVED** [review]

**Pull request merged**